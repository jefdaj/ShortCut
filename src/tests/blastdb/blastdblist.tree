.
|-- cache
|   |-- blastdb
|   |   `-- blastdbget
|   |       `-- dblist.txt
|   `-- list
|       |-- 399fc66708.txt
|       |-- 637edad317.txt
|       |-- 90aed1d1c9.txt
|       `-- d42c6637eb.txt
|-- exprs
|   |-- blastdblist
|   |   |-- 4c04f88f38.str.list
|   |   |-- 93c452e38f.str.list
|   |   |-- da67996088.str.list
|   |   `-- fa437bd7a3.str.list
|   |-- str
|   |   |-- 072c64781f.str -> ../../cache/list/d42c6637eb.txt
|   |   |-- 419a892f8b.str -> ../../cache/list/637edad317.txt
|   |   |-- 6634f1bc9a.str -> ../../cache/list/90aed1d1c9.txt
|   |   `-- 9d4568c009.str -> ../../cache/list/399fc66708.txt
|   `-- union
|       |-- 1d547713ed_7a898deeb0.str.list
|       |-- 3d6d858002_1c725bb84e.str.list
|       `-- 627a06b957_45f0826180.str.list
`-- vars
    |-- dbs.str.list -> ../exprs/blastdblist/4c04f88f38.str.list
    |-- human.str.list -> ../exprs/blastdblist/da67996088.str.list
    |-- nrs.str.list -> ../exprs/blastdblist/fa437bd7a3.str.list
    |-- proks.str.list -> ../exprs/blastdblist/93c452e38f.str.list
    `-- result -> ../exprs/union/627a06b957_45f0826180.str.list

9 directories, 21 files
