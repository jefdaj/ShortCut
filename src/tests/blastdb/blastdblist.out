[2017-10-16 20:40:22 - INFO] Communicating with BLAST server
[2017-10-16 20:40:22 - INFO] Communicating with BLAST server
[2017-10-16 20:40:22 - INFO] Communicating with BLAST server
[2017-10-16 20:40:22 - INFO] Communicating with BLAST server
["16SMicrobial",
 "cdd_delta",
 "env_nr",
 "env_nt",
 "est",
 "est_human",
 "est_mouse",
 "est_others",
 "gss",
 "gss_annot",
 "htgs",
 "human_genomic",
 "landmark",
 "nr",
 "nt",
 "other_genomic",
 "pataa",
 "patnt",
 "pdbaa",
 "pdbnt",
 "ref_prok_rep_genomes",
 "ref_viroids_rep_genomes",
 "ref_viruses_rep_genomes",
 "refseq_genomic",
 "refseq_protein",
 "refseq_rna",
 "refseqgene",
 "sts",
 "swissprot",
 "taxdb",
 "tsa_nr",
 "tsa_nt",
 "vector"]
